fem/loss_reinf.py

```diff
@@ -312,10 +328,16 @@ def match_desc_reward(points1projected, points2, desc1_int,
             quality = torch.ones(1).squeeze() * quality
             # reward = reward + geom_reward
 
-    # import pdb;pdb.set_trace()
-    # matches = numpy.stack([numpy.arange(len(points1projected))[wrong_id], k2_desc[wrong_id]])
-    # matches = numpy.stack([numpy.arange(len(points1projected)), k2_desc, (k2_desc != ind2).squeeze()])
-    # draw_m(img1, img2, matches, points, points2)
+    #from super_debug import draw_m
```

Comment:
maybe it's worth to remove redundant commented code?

Reply:
It's visualization code, i need it somewhere for future development..